pyproject.toml

```diff
@@ -25,7 +25,7 @@ dependencies = [
     "httpx",
     "numexpr",
     "numpy",
-    "morecantile>=3.1,<4.0",
+    "morecantile>=3.1,<5.0",
```

Comment:
for now only `4.0.0a0` has been released so the CI will still use morecantile==3.3